nestkernel/parameter.h

```diff
@@ -84,88 +93,6 @@ class Parameter
     return value( rng, nullptr );
```

Comment:
This and the previous `value()` method lack Doxygen comments and even the comment for the first does not explain the arguments.